### PR TITLE
Kernel updated to 5.2.0-m3

### PR DESCRIPTION
**Pull request by ashensw**

### CODE REVIEW

File: components/org.wso2.carbon.identity.event/pom.xml [hunk 1]
```diff
@@ -38,6 +38,10 @@
             <groupId>xerces</groupId>
             <artifactId>xercesImpl</artifactId>
         </dependency>
+        <!--<dependency>-->
+            <!--<groupId>commons-lang</groupId>-->
+            <!--<artifactId>commons-lang</artifactId>-->
+        <!--</dependency>-->
```
Comment (ruwanta) [inline components/org.wso2.carbon.identity.event/pom.xml]: Can remove the commented fragement

File: pom.xml [hunk 2]
```diff
@@ -93,6 +98,11 @@
                 <artifactId>snakeyaml</artifactId>
                 <version>${org.snakeyaml.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.commons</groupId>
+                <artifactId>commons-lang3</artifactId>
+                <version>${commons-lang3.version}</version>
+            </dependency>
```
Comment (ruwanta) [inline pom.xml]: lang3 dependency already present near line 81.

File: pom.xml [hunk 3]
```diff
@@ -73,6 +73,11 @@
                 <artifactId>commons-collections</artifactId>
                 <version>${commons-collections.version}</version>
             </dependency>
+            <!--<dependency>-->
+                <!--<groupId>commons-lang</groupId>-->
+                <!--<artifactId>commons-lang</artifactId>-->
+                <!--<version>${commons-lang.version}</version>-->
+            <!--</dependency>-->
```
Comment (ruwanta) [inline pom.xml]: can remove this fragment

File: pom.xml [hunk 4]
```diff
@@ -118,24 +128,25 @@
...
-        <carbon.feature.plugin.version>2.0.1</carbon.feature.plugin.version>
+        <carbon.feature.plugin.version>3.0.0</carbon.feature.plugin.version>
 
         <slf4j.version>1.7.1</slf4j.version>
         <slf4j.package.import.version>[1.7.1,2)</slf4j.package.import.version>
         <xerces.version>2.11.0</xerces.version>
         <xerces.package.import.version>[2.11,3)</xerces.package.import.version>
         <commons-lang3.version>3.4</commons-lang3.version>
         <commons-lang3.import.version.range>(3.4, 3.5]</commons-lang3.import.version.range>
+        <commons-lang.version>2.6</commons-lang.version>
```
Comment (ruwanta) [inline pom.xml]: We do not need this ?

File: components/org.wso2.carbon.identity.event/pom.xml (hunk 1 above)
Comment (ashensw) [inline components/org.wso2.carbon.identity.event/pom.xml]: Removed and updated the PR

File: pom.xml (hunk 3 above)
Comment (ashensw) [inline pom.xml]: Removed and updated the PR

File: pom.xml (hunk 2 above)
Comment (ashensw) [inline pom.xml]: Removed and updated the PR

File: pom.xml (hunk 4 above)
Comment (ashensw) [inline pom.xml]: Removed and updated the PR

File: feature/org.wso2.carbon.identity.common.feature/pom.xml [hunk 5]
```diff
@@ -43,6 +43,10 @@
             <artifactId>org.wso2.carbon.identity.event</artifactId>
             <version>${project.version}</version>
```
Comment (mefarazath) [inline feature/org.wso2.carbon.identity.common.feature/pom.xml]: Shall we remove the version from the dependency tag here and move it to the parent pom?

File: feature/org.wso2.carbon.identity.common.feature/pom.xml [hunk 6]
```diff
@@ -105,6 +109,10 @@
                                     <symbolicName>org.wso2.carbon.identity.event</symbolicName>
                                     <version>${project.version}</version>
```
Comment (mefarazath) [inline feature/org.wso2.carbon.identity.common.feature/pom.xml]: Shall we define a version property for identity.commons at root pom and use it here?

File: feature/org.wso2.carbon.identity.common.feature/pom.xml (hunk 5 above)
Comment (ashensw) [inline feature/org.wso2.carbon.identity.common.feature/pom.xml]: Updated the PR with the changes.

File: feature/org.wso2.carbon.identity.common.feature/pom.xml (hunk 6 above)
Comment (ashensw) [inline feature/org.wso2.carbon.identity.common.feature/pom.xml]: Updated the PR with the changes.

**Pull request merged**